static/sass/_pattern_deployment-table.scss

```diff
@@ -0,0 +1,8 @@
+@mixin anbox-p-deployment-table {
+  .table-cell {
+    @media (min-width: $breakpoint-medium) {
+      justify-content: center !important;
+      text-align: center !important;
+    }
```

Comment:
justify-content doesn't seem to do anything and the text-align seem to work without the `!important`
![image](https://user-images.githubusercontent.com/11927929/118645725-9298a780-b7df-11eb-9b61-f5bb9424566e.png)
So if it is no crucial I'd rather that we removed `!important`

Reply:
Fixed - thanks!